src/watch.ts

```diff
@@ -1,18 +1,24 @@
 import byline = require('byline');
 import request = require('request');
+import { Duplex } from 'stream';
```

Comment:
Needed to move `stream-buffers` from `devDependencies` to `dependencies` in `package.json` for this to work.